test.js

```diff
@@ -65,8 +65,8 @@ t.test('getMoonIllumination returns fraction and angle of moon\'s illuminated li
 t.test('getMoonTimes returns moon rise and set times', function (t) {
     var moonTimes = SunCalc.getMoonTimes(new Date('2013-03-04UTC'), lat, lng, true);
 
-    t.equal(moonTimes.rise.toUTCString(), 'Mon, 04 Mar 2013 23:57:55 GMT');
-    t.equal(moonTimes.set.toUTCString(), 'Mon, 04 Mar 2013 07:28:41 GMT');
+    t.equal(moonTimes.rise.toUTCString(), 'Mon, 04 Mar 2013 23:54:29 GMT');
+    t.equal(moonTimes.set.toUTCString(), 'Mon, 04 Mar 2013 07:47:58 GMT');
```

Comment:
I wonder if the new times match the info on websites like timeanddate.com, can you check?